### ISSUE TITLE
pod install fails with SwinjectAutoregistration

### ISSUE THREAD

**Issue by paulz** [Swinject/SwinjectAutoregistration#27]
trying to do `pod install`

to get `SwinjectAutoregistration`, after we already had `pod SwinjectStoryboard` cause this failure:

```
[!] Unable to satisfy the following requirements:

- `Swinject (= 2.1.0)` required by `Podfile.lock`
- `Swinject (~> 2.0)` required by `SwinjectStoryboard (1.1.1)`
- `Swinject (~> 2.0.0)` required by `SwinjectAutoregistration (2.0.0)`
```

`Podfile`:

```
    pod 'SwinjectStoryboard'
    pod 'SwinjectAutoregistration'
```

Does `SwinjectAutoregistration` works with `Swinject (= 2.1.0)`? or only with `2.0.0`?

**Comment by tkohout** [comment]
Hello paul, we're just in the process of updating the library to the newest version. It should be done in the latest release

**Comment by tkohout** [comment]
Fixed in Release [2.1.0](https://github.com/Swinject/SwinjectAutoregistration/releases/tag/2.1.0). Sorry for the delay
